src/main/resources/com/sonyericsson/jenkins/plugins/bfa/PluginImpl/config.jelly

```diff
@@ -111,5 +111,9 @@
                            default="${it.sodVariables.DEFAULT_SOD_COREPOOL_THREADS}"/>
             </f:entry>
         </f:section>
+        <f:entry title="${%Do not analyze aborted jobs}"
```

Comment:
normally the title is shorter than the description :) So the description does not add any value to the user being there. Either switch them or remove the description.